kurtosis-devnet/tests/eigenda/failover_test.go

```diff
@@ -1,34 +1,24 @@
-package eigenda_test
+// This package consists of tests against a kurtosis enclave running the eigenda-devnet, with proxy in memstore.
+// All tests in this package should be tests that explicitly require memstore, such as failover test.
```

Comment:
Your comment here says that all tests in the package should use memstore, but there are tests in the same package (in `batcher_logs_test.go`) that use holesky

Reply:
Oh yeah that was old; I did a refactor to use test suffixes to target memstore vs holesky, instead of using packages. Removed the commend in [03b82ba](https://github.com/Layr-Labs/optimism/pull/39/commits/03b82ba98f2cedbac529f57b85968830707cf70c)